package/src/utils/utils.ts

```diff
@@ -82,6 +84,16 @@ export const getIndicatorTypeForFileState = (
   return indicatorMap[fileState];
 };
 
+export const isBlockedMessage = <
+  StreamChatGenerics extends DefaultStreamChatGenerics = DefaultStreamChatGenerics,
+>(
+  message: MessageType<StreamChatGenerics> | TableRowJoinedUser<'messages'>,
+) => {
+  // The only indicator for the blocked message is its message type is error and that the message text contains "Message was blocked by moderation policies".
+  const pattern = /\bMessage was blocked by moderation policies\b/;
```

Comment:
```suggestion
  const pattern = /\bMessage was blocked by moderation policies\b/;
```

even if the local language is different... we get the same message?

Reply:
Do we send the backend our local language?